### PR TITLE
Reorder and add descriptions of make targets

### PR DESCRIPTION
**Pull request by guicassolato**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -297,35 +295,33 @@ bundle-build: ## Build the bundle image.
...
+prepare-release: ## Prepares a release: create build info file, generate manifests, OLM bundle and Helm chart.
 	$(MAKE) create-build-file
 	$(MAKE) set-authorino-default-image
 	$(MAKE) set-replaces-directive
 	$(MAKE) manifests bundle VERSION=$(VERSION) AUTHORINO_VERSION=$(AUTHORINO_VERSION)
 	$(MAKE) helm-build VERSION=$(VERSION) AUTHORINO_VERSION=$(AUTHORINO_VERSION)
 
-# A comma-separated list of bundle images (e.g. make catalog-build BUNDLE_IMGS=example.com/operator-bundle:v0.1.0,example.com/operator-bundle:v0.2.0).
-# These images MUST exist in a registry and be pull-able.
-BUNDLE_IMGS ?= $(BUNDLE_IMG)
```
Comment (guicassolato) [inline Makefile:327]: No longer used since #201.

File: Makefile [hunk 2]
```diff
@@ -3,6 +3,8 @@ SHELL = /bin/bash
 
 MKFILE_PATH := $(abspath $(lastword $(MAKEFILE_LIST)))
 
+PROJECT_DIR := $(shell dirname $(abspath $(lastword $(MAKEFILE_LIST))))
```
Comment (guicassolato) [inline Makefile:6]: Needs to be defined before the `##@ Tools` section.

File: Makefile [hunk 3]
```diff
@@ -297,35 +295,33 @@ bundle-build: ## Build the bundle image.
 bundle-push: ## Push the bundle image.
 	$(MAKE) docker-push OPERATOR_IMAGE=$(BUNDLE_IMG)
 
+##@ Release
+
 .PHONY: create-build-file
-create-build-file: $(YQ)
+create-build-file: $(YQ) ## Created the build info file.
```
Comment (KevFan) [inline Makefile:301]:
```suggestion
create-build-file: $(YQ) ## Creates the build info file.
```

File: Makefile [hunk 4]
```diff
@@ -297,35 +295,33 @@ bundle-build: ## Build the bundle image.
...
 .PHONY: set-authorino-default-image
-set-authorino-default-image: $(YQ)
+set-authorino-default-image: $(YQ) ## Sets the default Authorino image in the build file.
 	@if [ "$(AUTHORINO_VERSION)" != "latest" ]; then\
 		V="$(DEFAULT_REGISTRY)/$(DEFAULT_ORG)/authorino:$(AUTHORINO_IMAGE_TAG)" $(YQ) eval '.config.authorinoImage = strenv(V)' -i $(BUILD_CONFIG_FILE); \
 	fi
 
 .PHONY: set-replaces-directive
-set-replaces-directive: $(YQ)
+set-replaces-directive: $(YQ) ## Sets the calue for the OLM replaces directive in the build file.
```
Comment (KevFan) [inline Makefile:311]:
```suggestion
set-replaces-directive: $(YQ) ## Sets the value for the OLM replaces directive in the build file.
```

**Review by KevFan — APPROVED** [review]
Looks good to me 👍

**Pull request merged**